File: cpp/fact.cpp
#include<iostream>
using namespace std;

int main(){
int n;
cout<<"Enter number";
cin>>n;
int fact = 1;
for(int i=n;i>0;i--){
	fact = fact* i;
}
cout<<"factorial of "<<n<<" is "<<fact<<endl;
}

File: 0_Assignment/1/fact.java
import java.util.*;
class fact{
	public static void main(String a[]){
		Scanner sc = new Scanner(System.in);	
		System.out.println("Enter Number :");
		int num = sc.nextInt();
		int fact = 1;
		for(int i=num;i>0;i--){
			fact = fact* i;
		}
		System.out.println("Factorial of "+num+" is "+fact);
	}
}
